src/styles/block/card.scss

```
@@ -0,0 +1,111 @@
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $bord_color;
+  border-radius: 5px;
+  margin-bottom: 50px;
+
+  &__image {
+    display: flex;
+    height: 134px;
+    width: 160px;
+    margin: 0 3px 50px;
```

```suggestion
    margin: 0 3px 40px;
```
it's 40px in figma. why set it to 50?